t/page/term_table.rb

```diff
@@ -125,34 +125,32 @@
         af.contacts.first.link.must_equal 'mailto:angela.fichtinger@parlament.gv.at'
       end
 
-      it 'has two identifiers' do
-        af.identifiers.count.must_equal 2
-      end
-
-      it 'has Wikidata' do
-        af.identifiers.first.type.must_equal 'wikidata'
-        af.identifiers.first.value.must_equal 'Q15783437'
-        af.identifiers.first.link.must_equal 'https://www.wikidata.org/wiki/Q15783437'
-      end
-
-      it 'has a parliament identifier' do
-        af.identifiers.last.type.must_equal 'parlaments_at'
-        af.identifiers.last.value.must_equal '83146'
-        af.identifiers.last.link.must_equal nil
-      end
-
-      it 'has up to 5 identifiers' do
-        harald = subject.people.find { |p| p.name == 'Dr. Harald Walser' }
-        harald.identifiers.size.must_equal 5
-      end
-
-      it 'has identifiers ordered by term frequency' do
-        harald = subject.people.find { |p| p.name == 'Dr. Harald Walser' }
-        harald.identifiers[0].type.must_equal 'wikidata'
-        harald.identifiers[1].type.must_equal 'parlaments_at'
-        harald.identifiers[2].type.must_equal 'viaf'
-        harald.identifiers[3].type.must_equal 'gnd'
-        harald.identifiers[4].type.must_equal 'lcauth'
+      describe 'identifiers' do
+        let(:maria) { subject.people.find { |p| p.name == 'Mag. Dr. Maria Theresia Fekter' } }
```

Comment:
If you're setting up a series of tests for a different person, those should probably be out one level from this — not embedded in the middle of the Angela Fichtinger tests.